modules/sara_r4/cellular_r4_api.c

```diff
@@ -949,6 +1263,15 @@ CellularError_t Cellular_SocketConnect( CellularHandle_t cellularHandle,
             /* Revert the state to allocated state. */
             socketHandle->socketState = SOCKETSTATE_ALLOCATED;
         }
+        else if( socketHandle->socketProtocol == CELLULAR_SOCKET_PROTOCOL_UDP )
+        {
+            /* No need to wait for other URC response for UDP. */
```

Comment:
Suggest to update the doxygen document as well to describe the socket state of UDP. If new state is required, we can add the socket state.

Reply:
Added.